book.toml

```
@@ -8,8 +8,9 @@ src = "src"
 [build]
 create-missing = true # This is kept for convenience, but CI sets it to false
 use-default-preprocessors = false
+extra-watch-dirs = ["po"]
 
-[preprocessor.links]
+[preprocessor.gettext]
```

```suggestion
[preprocessor.links]

[preprocessor.gettext]
```
The built-in `links` preproc is required to expand `{{#include}}`.

That said, this presents an interesting question: in which order should these two run? I.e. should the `include`d text be localised? I'd say no, and then we should add a `before = ["links"]` line to the `[preprocessor.gettext]` table.

---

Agree, I changed as suggested